blog-post.md

```diff
@@ -0,0 +1,68 @@
+The OpenAPI Initiative is pleased to announce a patch release of the 3.0 and 3.1 OpenAPI specifications.
+In patch releases, no changes are made to the way that APIs are described, but the specification wording itself contains many updates, expansions, and clarifications where previous the points may have been unclear or not covered.
+Think of this release as the "Words Mean Things" edition.
+
+## Released versions
+
+3.1.1 is the newest and recommended version of OpenAPI.
+If you are starting a new project today, or have the option to upgrade, this is your target version.
+Tooling that supports 3.1.0 is expected to work without problems on 3.1.1 since the patch releases don't contain structure changes.
+
+3.0.4 is an additional release on the 3.0 branch to incorporate the improved wording to the 3.0 branch of the specification where the changes applied there.
+It is expected that 3.0.4 will be the final release in the 3.0.x line.
+
+## Summary of changes
+
+The releases include as many explanations, clarifications and expanded sections as we could manage, driven mostly by the questions and comments we get from the users and tools creators of OpenAPI.
+The highlights include a lot of new content to expand on existing content and reduce ambiguity.
+The sections regarding parameters, encoding and schemas have had significant updates and expansion to cover more cases.
+You will also find some security clarifications and a whole new "Security Considerations" section has been added.
+
+Look out for additional appendices with some great explanations that support the additions to the main body of the specification.
+We added a great collection of new content sections and appendix entries about handling data including data types, serialization and encoding.
+In 3.1, there is more information about parsing documents and resolving references since the adoption of JSON Schema.
```

Comment:
```suggestion
In 3.1, there is more information about parsing documents and resolving references since the adoption of full JSON Schema compatibility.
```